gene2phenotype_project/gene2phenotype_app/urls.py

```diff
@@ -81,5 +81,8 @@ def perform_create(self, serializer):
     path('logout/', views.LogOutView.as_view(), name='logout'),
     path('token/refresh/', views.CustomTokenRefreshView.as_view(), name='token_refresh'),
 
+    #meta information
+    path("ref_info/", views.MetaView.as_view(), name="get_reference_data")
```

Comment:
`ref_info` could be renamed to `reference_data`.

Reply:
Updated!